imap_processing/mag/mag_cdf_attrs.py

```diff
@@ -118,3 +176,80 @@
     display_type="time_series",
     label_axis="Magnetic field vector",
 )
+
+# Catdesc (<80 chars), Fieldnam (<30 chars)
+
+catdesc_fieldname_l0 = {
+    # TODO: Don't include PUS_SPARE1?
+    "PUS_SPARE1": ["Spare header from ESA Standard", "Spare header"],
```

Comment:
could you change these list to dictionary as well? I was wondering in above code what those 0 or 1 index mean. Having those key referenced may be much easier to follow without looking up here.
```suggestion
    "PUS_SPARE1": { "catdesc": "Spare header from ESA Standard", "fieldnam": "Spare header"},
```

Reply:
I like how you've organized these!